### DISCUSSION TITLE
Open GitHub Discussion listing of repo

### DISCUSSION THREAD

**Discussion by lannonbr** [johnlindquist/kit#96] [Share]
Did a quick remix on the `chrome-tab` script @johnlindquist made so I could quickly navigate to the discussions tab of any repo.

```js
// Description: Open the Discussions pane of a GitHub repo
// Author: Benjamin Lannon
// Twitter: @lannonbr

let { focusTab } = await kit('chrome')
let repo = await arg('Enter repo (ex: johnlindquist/kit):')

focusTab(`https://github.com/${repo}/discussions`)
```

**Comment by johnlindquist** [comment]
@lannonbr I've been using this variation about 100 times an hour ⚡️ It reads in any existing bookmarks and merges it with currently open tabs. Adding a bookmark in Chrome is cmd+D, so I can easily add it to the list without any thought:

https://scriptkit.app/scripts/johnlindquist/chrome-tab-advanced
